### DISCUSSION TITLE
WiFi Dongle no open ports?

### DISCUSSION THREAD

**Discussion by traxata** [celsworth/lxp-bridge#272] [Q&A] [answered]
Hello, I'm wondering if I'm missing something or if something has changed on new system installs.

I wasn't sure what version of the dongle I'd receive and I'm not entirely sure which I received to be honest!. I have a 'squirrel pod' system so have 2x dongles and each inverter at the moment has a single uhome 2400 battery on it. Both the batteries and the inverters / dongles are connected to the wifi and I can see all 4 devices listed as espressif devices. I've run an open port scan on my entire internal network and on individual IPs but can't talk to configure them... 
Any help would be fantastic as I'd very much like to stop them from talking to china if I can :)

**Comment by nickjf20** [comment, marked as answer]
The newer dongles don't have a web interface however should still work with the default settings listed on the wiki.

If you disable internet access to the dongle you will need to setup 'heartbeats' in lxp-bridge.

**Reply by traxata** [reply]
Ah wonderful. I had a misconfig in MQTT so wasn't seeing all the entities. I am now without having to adjust the dongle ports.
I'll leave the heartbeats disabled and keep the internet access until I'm certain that it all works I think.

I do have a different question that may partly be why I wasn't seeing anything before...

On a 2+ inverter parallel system, if the wrong IP address is enterered in the config with the dongle ID / serial numbers input in the config will I still be able to read all the data but not be able to send any commands back to the inverters? 

I am trying to set up a simple charge check. See attached image or:
If Ohme Home Pro is Charging "car" then - turn on AC charge on inverter 1, turn on AC charge on inverter 2. 

A separate automation doing the reverse when ohme charger stops charging "car"
Currently that automation does not triggle AC charge

![ohme car charge setup](https://github.com/user-attachments/assets/198ee336-bcb0-4ac3-ad07-ffb25eac4d0a)

**Reply by traxata** [reply]
Also ignore the above. I've realised I'm toggling the AC CHARGE enable / disable flag instead of telling it to charge. I was basically looking for a force charge like the button on the web ui... but that just puts in a 1 hour charge slot!
